go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -1779,8 +1782,7 @@ func (fbo *folderBranchOps) setHeadLocked(
 	}
 
 	if ct == mdCommit {
-		fbo.doneWg.Add(1)
-		go fbo.commitFlushedMD(md, fbo.latestMergedUpdated)
+		fbo.goTracked(func() { fbo.commitFlushedMD(md, fbo.latestMergedUpdated) })
```

Comment:
This introduces a new data race -- `fbo.latestMergedUpdated` needs to be read while holding the lock, before the goroutine is made.